scapy/contrib/ife.uts

```diff
@@ -0,0 +1,27 @@
+% IFE test campaign
+
+#
+# execute test:
+# > test/run_tests -P "load_contrib('ife')" -t scapy/contrib/ife.uts
+#
+
++ Basic layer handling
+= build basic IFE frames
+
+frm = Ether()/IFE(tlvs=[IFESKBMark(), IFETCIndex()])
+frm = frm.build()
+frm = Ether(frm)
```

Comment:
Could you enhance those a tiny bit
```suggestion
frm = Ether(frm)
assert IFE in frm
assert frm[IFE].tlvs[0].type == 1
assert frm[IFE].tlvs[0].type == 5
```

Reply:
That only works on tlvs[0] and I don't know why. When I do:
print(frm[IFE].tlvs)

[<IFETlv  type=SKBMark length=8 value=3 |<Raw  load='\x00\x05\x00\x06\x00\x05\x00\x00' |>>]

Somehow the tail get's "Raw load"... I also changed the second to tlvs[1], but I can't access it. I can see the TCIndex there (I changed a little bit the value string to make an assert on it)

Reply:
You need to add an `extract_padding`. See https://stackoverflow.com/a/8101584/5459467